README.rst

```diff
@@ -447,6 +447,62 @@ of the url, this will enable you to point all the API urls to
 your openwisp-network-topology API server's domain,
 example value: ``https://mytopology.myapp.com``.
 
+``Rest API``
+------------
+
+``Live documentation``
+^^^^^^^^^^^^^^^^^^^^^^
+
+.. image:: docs/api-doc.png
+
+A general live API documentation (following the OpenAPI specification) at ``/api/v1/docs/``.
+
+``Browsable web interface``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. image:: docs/api-ui.png
+
+Additionally, opening any of the endpoints `listed below <#list-of-endpoints>`_
+directly in the browser will show the `browsable API interface of Django-REST-Framework
+<https://www.django-rest-framework.org/topics/browsable-api/>`_,
+which makes it even easier to find out the details of each endpoint.
+
+``List of endpoints``
+^^^^^^^^^^^^^^^^^^^^^
+
+Since the detailed explanation is contained in the `Live documentation <#live-documentation>`_
+and in the `Browsable web page <#browsable-web-interface>`_ of each point,
+here we'll provide just a list of the available endpoints,
+for further information please open the URL of the endpoint in your browser.
+
+``List topologies``
+###################
+
+.. code-block:: text
+
+    GET /api/v1/topology/
+
+``Detail of a topology``
+########################
+
+.. code-block:: text
+
+    GET /api/v1/topology/{id}/
+
+``View history of a topology``
+##############################
+
+.. code-block:: text
+
+    GET /api/v1/topology/{id}/history/
```

Comment:
here it would be nice to show information about how to go back in time and see previous topology snapshots, also mention (and link to) the `save_snapshot` section, indicating that snapshot should be saved periodically for this feature in order to work.